packages/tailwindcss/src/apply.ts

```diff
@@ -161,11 +161,21 @@ export function substituteAtApply(ast: AstNode[], designSystem: DesignSystem) {
       {
         // Parse the candidates to an AST that we can replace the `@apply` rule
         // with.
-        let candidateAst = compileCandidates(candidates, designSystem, {
-          onInvalidCandidate: (candidate) => {
-            throw new Error(`Cannot apply unknown utility class: ${candidate}`)
-          },
-        }).astNodes
+        let candidateAst
+        try {
+          candidateAst = compileCandidates(candidates, designSystem, {
+            onInvalidCandidate: (candidate) => {
+              throw new Error(`Cannot apply unknown utility class: ${candidate}`)
```

Comment:
Instead of wrapping a try catch can't we just remove this throw and replace it with a comment?

The returned AST should be empty I think?

Reply:
Ah wait no, it'll have _some_ content in the case that there are some valid and some invalid 🤔 

I guess partial application of an `@apply` might be hard to track down so ig this is fine